components/Header.tsx

```diff
@@ -8,6 +9,9 @@ function Header() {
         <Typography variant="h5" component="h1" sx={{ flexGrow: 1 }}>
           Yummy Rater
         </Typography>
+        {/* <div>
+          <Navigation />
+        </div> */}
```

Comment:
As we're not going to use the navigation in the header, could you please remove it here?